website/src/docs/migration-guides.md

```diff
@@ -92,27 +92,29 @@ If you are using CommonJS, you might need to add some tooling for everything to
 
 See the [Robodog migration guide](#Migrate-from-Robodog-to-Uppy-plugins).
 
-### `@uppy/core`
+### Uppy packages
 
-#### Remove `AggregateError` polyfill.
+#### `@uppy/core`
```

Comment:
I think I prefer not increasing the amount of headers, I made it this way as it looked good on our website, header levels too deep don't look too nice with the current website. I think it's fine to just have "Known issues" below the packages.

Reply:
`@uppy/core` is an h3, but the next packages are h4, that doesn't feel right.

Reply:
No, only `@uppy/core` is h3: https://github.com/transloadit/uppy/blob/feee2afd63d18873992ec8c6fe9fbb17d4833cea/website/src/docs/migration-guides.md#L95

`@uppy/transloadit` is h4: https://github.com/transloadit/uppy/blob/feee2afd63d18873992ec8c6fe9fbb17d4833cea/website/src/docs/migration-guides.md#L121

`@uppy/tus/`, `@uppy/aws-s3`, `@uppy/xhr-upload`h4 as well: https://github.com/transloadit/uppy/blob/feee2afd63d18873992ec8c6fe9fbb17d4833cea/website/src/docs/migration-guides.md#L151

etc.: https://github.com/transloadit/uppy/blob/feee2afd63d18873992ec8c6fe9fbb17d4833cea/website/src/docs/migration-guides.md#L157 https://github.com/transloadit/uppy/blob/feee2afd63d18873992ec8c6fe9fbb17d4833cea/website/src/docs/migration-guides.md#L171 https://github.com/transloadit/uppy/blob/feee2afd63d18873992ec8c6fe9fbb17d4833cea/website/src/docs/migration-guides.md#L177 https://github.com/transloadit/uppy/blob/feee2afd63d18873992ec8c6fe9fbb17d4833cea/website/src/docs/migration-guides.md#L183 https://github.com/transloadit/uppy/blob/feee2afd63d18873992ec8c6fe9fbb17d4833cea/website/src/docs/migration-guides.md#L189

Reply:
Can we instead make them H3? For reasons mentioned above

Reply:
Sure, that's how I did at first. However because H3s appear on the side nav panel, I thought that would be too much.

Reply:
Alright, whatever feels/looks best then, I don't have this running locally 👍